### DISCUSSION TITLE
RSA certificate generation

### DISCUSSION THREAD

**Discussion by Titanexx** [Titanexx/NightWriter#4] [General]
Let's continue here the [discussion](https://github.com/Titanexx/NightWriter/pull/1#issuecomment-887322977).

@noraj, `it could be nice to let the user provide one in the config`, it's on my roadmap. 

`the keygen was never ending`, it's sound more like an error for me. It's quite fast (10-15sec) on my computer.
The error management must be improved.

**Comment by noraj** [comment]
> It's quite fast (10-15sec) on my computer.

Weird I waited 2min and still nothing, I'll retry later.

**Comment by noraj** [comment]
It seems the script never loads

![image](https://user-images.githubusercontent.com/16578570/127904748-1c7bc862-8232-4015-8e2e-ba53666d9558.png)

`/forge/prime.worker.js`

```html
<!DOCTYPE html>
<html class="dark h-full" lang="en">
<head>
  <meta charset="UTF-8">
  <link rel="icon" href="/favicon.ico" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Vite App</title>
<script type="module" crossorigin src="/assets/index.08910e08.js"></script>
  <link rel="modulepreload" href="/assets/vendor.e5f0a4dc.js">
  <link rel="stylesheet" href="/assets/index.5712825c.css">
</head>
<body class="h-full bg-primary text-primary break-words flex flex-col">
  <div id="app" class="flex-auto min-h-0 overflow-y-auto"></div>
  
  <script>
    if (localStorage.theme === 'dark' || (!'theme' in localStorage && window.matchMedia('(prefers-color-scheme: dark)').matches)) {
      document.querySelector('html').classList.add('dark')
    } else if (localStorage.theme === 'dark') {
       document.querySelector('html').classList.add('dark')
    }
  </script>
</body>
</html>
```

It don't seems to be the correct script.
